code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -130,3 +130,6 @@
 
 /obj/item/encryptionkey/secbot
 	channels = list(RADIO_CHANNEL_AI_PRIVATE = 1, RADIO_CHANNEL_SECURITY = 1)
+
+/obj/item/encryptionkey/antenna //for antenna mutation and it's unique demands. hopefully it wont be pulled out somehow.
```

Comment:
```suggestion
/obj/item/encryptionkey/antenna //for antenna mutation and it's unique demands. hopefully, it won't be pulled out somehow.
```

Reply:
its a comment, nobody cares about grammar

Reply:
> its a comment, nobody cares about grammar

I just read it and thought, "who the hell doesn't know their won't spellings" enough to want to check who the author is, so whether you like it or not, people care, and it DOES make you look bad.

There isn't anything wrong with looking illiterate, so it's up to you. I'm not forcing you to take the suggestion.